apps/music/js/metadata/ogg.js

```diff
@@ -95,109 +129,199 @@ var OggMetadata = (function() {
             break;
           }
           if (!valid) {
-            reject('malformed ogg comment packet');
-            return;
+            reject(new Error('malformed ogg comment packet'));
           }
 
-          readAllComments(fullpage, metadata);
+          var vendor_string_length = fullpage.readUnsignedInt(true);
+          fullpage.advance(vendor_string_length);// skip libvorbis vendor string
```

Comment:
Wait... the linter actually demands inline comments not have a space before them? That's gross.